### ISSUE TITLE
No benefit of pre-trained weight: Incremental classes training

### ISSUE THREAD

**Issue by satpalsr** [open-mmlab/mmdetection#10390]
I found no benefit in using custom pre-trained weight to start training.
Example: I start from pre-trained weights from [retinanet configs page](https://github.com/open-mmlab/mmdetection/tree/main/configs/retinanet) and train retinanet model for 100 classes.

Then I use 100 classes trained model weight as pre-train weight (load_from) to train 130 classes (30 new classes added).
This training loss looks similar to when I start with pre-trained retinanet weight.
So, I see no benefit of using my custom pre-trained weight.

Ideally, the loss should start with a lower value when using 100 classes model weight to start training with.
